src/OpenTelemetry.Instrumentation.AspNet/Implementation/AspNetInstrumentationEventSource.cs

```diff
@@ -24,50 +24,50 @@ namespace OpenTelemetry.Instrumentation.AspNet.Implementation
     /// EventSource events emitted from the project.
     /// </summary>
     [EventSource(Name = "OpenTelemetry-Instrumentation-AspNet")]
-    internal class AspNetInstrumentationEventSource : EventSource
+    internal sealed class AspNetInstrumentationEventSource : EventSource
     {
         public static AspNetInstrumentationEventSource Log = new AspNetInstrumentationEventSource();
 
         [NonEvent]
-        public void RequestFilterException(Exception ex)
+        public void RequestFilterException(string operationName, Exception ex)
         {
             if (this.IsEnabled(EventLevel.Error, EventKeywords.All))
             {
-                this.RequestFilterException(ex.ToInvariantString());
+                this.RequestFilterException(operationName, ex.ToInvariantString());
             }
         }
 
-        [Event(1, Message = "Payload is NULL in event '{1}' from handler '{0}', span will not be recorded.", Level = EventLevel.Warning)]
-        public void NullPayload(string handlerName, string eventName)
+        [NonEvent]
+        public void EnrichmentException(string eventName, Exception ex)
         {
-            this.WriteEvent(1, handlerName, eventName);
+            if (this.IsEnabled(EventLevel.Error, EventKeywords.All))
+            {
+                this.EnrichmentException(eventName, ex.ToInvariantString());
+            }
         }
 
-        [Event(2, Message = "Request is filtered out.", Level = EventLevel.Verbose)]
-        public void RequestIsFilteredOut(string eventName)
+        [Event(1, Message = "Payload is NULL in event '{1}' from handler '{0}', span will not be recorded.", Level = EventLevel.Warning)]
```

Comment:
Curious - is there a reason put to have `{1}` before `{0}`?

Reply:
Null payload actually isn't possible anymore so I just removed the log.